app/src/main/res/values/strings.xml

```diff
@@ -304,55 +304,55 @@
-        \n\n2- Wait until \'%s\' appears on the Wi-Fi list
-        \n\n3- Pick it (%s) and paste the password (%s)
-        \n\n4- Return here after successful connection
+    <string name="wifi_connection_notice">1- Open Wi-Fi settings.
+        \n\n2- Wait until \'%s\' appears on the Wi-Fi list.
+        \n\n3- Pick it (%s) and paste the password (%s).
+        \n\n4- Return here after connecting.
     </string>
-    <string name="copy_password_and_open_settings"><![CDATA[Copy Password & Open Settings]]></string>
+    <string name="copy_password_and_open_settings"><![CDATA[Copy Password adn Open Settings]]></string>
     <string name="wifi_password">Wi-Fi Password</string>
     <string name="notifications_instructive">Instructive Notifications</string>
     <string name="connect_to_wifi_notice">Connect to \"%s\"</string>
     <string name="enter_wifi_password_notice">Paste the password \"%s\"</string>
     <string name="not_connected_yet_notice">Not connected yet</string>
-    <string name="location_service_disabled_notice">Location Service is disabled</string>
-    <string name="wifi_disabled_notice">Wi-Fi is disabled</string>
+    <string name="location_service_disabled_notice">Location service off</string>
+    <string name="wifi_disabled_notice">Wi-Fi off</string>
     <string name="write_system_settings_permission_required_notice">Needs permission to modify system settings</string>
-    <string name="connecting_to_client">Connecting to Client</string>
+    <string name="connecting_to_client">Connecting to Client…</string>
     <string name="client_offline_notice">The remote is unavailable</string>
     <string name="matching_client_question">Is this the client you are looking for?</string>
     <string name="verify_client">Verify Client</string>
     <string name="client_details">Client Details</string>
     <string name="client_model">Model</string>
     <string name="percent_sign" translatable="false">%</string>
-    <string name="waiting_receiver_approval_notice">Waiting for the approval of receiver</string>
+    <string name="waiting_receiver_approval_notice">Awaiting recipient approval</string>
     <string name="incoming">Incoming</string>
     <string name="outgoing">Outgoing</string>
-    <string name="error_communication_unknown">An unknown communication error occurred</string>
-    <string name="error_client_no_address">The client has no address (try manually connecting)</string>
-    <string name="error_protocol_unknown">An unknown protocol error occurred</string>
+    <string name="error_communication_unknown">Unknown communication error</string>
+    <string name="error_client_no_address">The client has no address (try connecting manually)</string>
+    <string name="error_protocol_unknown">Unknown protocol error</string>
     <string name="completed">Completed</string>
-    <string name="run_in_background">Run in Background</string>
+    <string name="run_in_background">Run in Background…</string>
```

Comment:
This is a button in the quick settings tile, so this change is not suitable.

![Screenshot_20210828-163429_TShot_DEBUG](https://user-images.githubusercontent.com/10258162/131219719-d9ea7e96-adc5-4d29-a899-1b9e3a0ed983.png)

Reply:
```suggestion
    <string name="run_in_background">Run in Background</string>
```